### DISCUSSION TITLE
Visual Snow Relief Overlay main discussion

### DISCUSSION THREAD

**Discussion by belvederef** [belvederef/visual-snow-relief-overlay#1] [General]
## 👋 Welcome!
Thank you for your interest in the Visual Snow Relief Overlay project. In this discussion you can:
  * Get involved and offer to help.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  
Remember that this is a community we
  build together 💪.

**Comment by belvederef** [comment]
I want to thank all the people that are getting involved in the project. Overwhelming response from all, let's see where we go from here!

**Reply by MatteoGheza** [reply]
I'm trying to run electron-builder in Github Actions.
Tomorrow I'll open a PR.

**Reply by belvederef** [reply]
Hey Matteo, thanks for joining. That'd be an amazing feature to have.

**Reply by MatteoGheza** [reply]
#8 Initial Github Actions support

**Comment by emmalovesgreys** [comment]
Hi, I have a MacBook air and I cannot find the download link for the overlay. I am not very technically talented!

**Reply by belvederef** [reply]
Hi Emma! If you go to the project page and scroll down, you will see I have included some download instructions (the section is called "Download", [here is a link to it](https://github.com/belvederef/visual-snow-relief-overlay#download)) that guides you through choosing the right version for your operating system. 

Once you know the version you need, you can go to the [release page](https://github.com/belvederef/visual-snow-relief-overlay/releases). From there, pick the topmost option (it is the most recent) and you will see there is a list of files under "assets". The actual programs are the ones starting with "VS-Relief-Overlay". When you have the right one for your system, you will just need to extract or install it, depending on the version you chose, and the overlay will be there!

Let me know if everything is clear

**Comment by kayleefrogs** [comment]
Hi, using this on Windows 11 and am having trouble with my two monitor setup.  When I run it, it works on one screen, but I can't do anything on the second.  When I use win+shift+arrow to switch the program to the second monitor, it still won't do anything.  I can't change settings or interact with the window at all.  I am very computer illiterate but would love to be able to use this during the long hours of med school studying (:
